doc/lua_api.txt

```diff
@@ -6727,10 +6727,29 @@ object you are working with still exists.
     object.
 * `set_detach()`
 * `set_bone_position([bone, position, rotation])`
-    * `bone`: string. Default is `""`, the root bone
-    * `position`: `{x=num, y=num, z=num}`, relative, `default {x=0, y=0, z=0}`
-    * `rotation`: `{x=num, y=num, z=num}`, default `{x=0, y=0, z=0}`
+	* Shorthand for `set_bone_override(bone, {position = {x = position.x, y = position.y, z = position.z, absolute = true}, rotation = {x = rotation.x, y = rotation.y, z = rotation.z, absolute = true}})`
+	* Supports older servers & clients (< 5.6)
+	* **Deprecated**, use `set_bone_override` instead
 * `get_bone_position(bone)`: returns position and rotation of the bone
+	* Shorthand for `get_bone_override(bone).position.vector, get_bone_override(bone).rotation.vector`
+	* **Deprecated**, use `get_bone_override` instead
+* `set_bone_override(bone, override)`
+    * `bone`: string
+    * `override`: `{ position = property, rotation = property, scale = property }` or `nil`
```

Comment:
+4 spaces for each indent, especially for the `absolute` part above to easier distinguish the sub-lists.